src/App.js

```diff
@@ -4,6 +4,7 @@ import TopBar from './TopBar';
 import Content from './Content';
 import { useAuthState, AuthStatus } from './Auth';
 import Login from './Login';
+import PWReset from './ResetPassword';
```

Comment:
A useful convention for default exports is to import them with the same name as the file:

`import ResetPassword from './ResetPassword';`